index.js

```diff
@@ -161,7 +150,7 @@ WuiDom.prototype.removeChild = function (child) {
 
 	this.rootElement.removeChild(child.rootElement);
 	this._childrenList.splice(siblingIndex, 1);
-	if (this._childrenMap.hasOwnProperty(child._name)) {
+	if (child._name in this._childrenMap) {
```

Comment:
Right. So if the name of my child is "__proto__" it will return `true`.
But in that case we are talking about a different problem here, no?